### DISCUSSION TITLE
[05/26] 피어세션 일지

### DISCUSSION THREAD

**Discussion by scarlettJin** [bcaitech1/p4-dkt-lastffang#18] [Peer Session]
### **홍지민 조교님 소개**
- kaist에서 번역, text, 시각화 연구하시는 분!

### **멘토링 어떻게 진행하면 좋을지 방향 정해보기**
- 질문을 정리해서 조교님께 미리 전달하고  조교님이 답변 해주는 형태로 진행하도록 함

### **금일 멘토링** 
- 다음주에 nc soft에서 고객 이탈 예측 task 에 대한 시각화 자료 설명해주시기로 함
- 일정: 매주 화요일 4시반에 참가하시기로 함

### **각자 한 일 공유시간(자유롭게 추가해주세요)**
@배새봄
wandb sweep

@정근영
trainer 등 소스 분석 자료 공유

@김도윤
EDA 관련 문의 및 공유

@박건희
이전 정형 프로젝트 코드 올려주심

@나윤석
LGBM 에서는 feature engineering 을 얼마나 잘하냐고 중요함
각 feature들의 통계값을 다 만들어두고, 트리계열 BOOSTING 모델을 돌려서 중요한 FEATURE를 찾는 것이 중요함

@김진홍
hidden layer 사이즈 별 테스트 - 256 가 가장 좋았음
위 조건에 batch 사이즈는 16이 가장 좋았음.

### 리더보드 티켓 사용 Rule 
- 우선 각자 하나씩 사용하고, 나머지는 선착순으로 사용 가능
- 사용 후 카톡에 사용여부 공유

(빠진 내용이나 틀린 사항 있으면 자유롭게 추가/변경해주세요~)

**Comment by GY-Jeong** [comment]
베이스라인 코드 적어놓은건 저도 한번 보고 정리한거라 틀린 내용이 있을수 있어요!
혹시 다른 내용 있으면 알려주세요~!!!
